src/client_side_reply.cc

```diff
@@ -1486,15 +1487,12 @@ clientReplyContext::identifyStoreObject()
 
     // client sent CC:no-cache or some other condition has been
     // encountered which prevents delivering a public/cached object.
-    // XXX: The above text does not match the condition below. It might describe
-    // the opposite condition, but the condition itself should be adjusted
-    // (e.g., to honor flags.noCache in cache manager requests).
```

Comment:
If we cover in this PR only CacheManager requests - should we still keep this XXX? If there are no other cases to fix (we aware of, such as cache_digest) then we do not need it, I think.

Reply:
The goal of this PR is to fully address that XXX. Thus, this PR should (among other changes) remove that XXX comment. This change is correct.